src/panels/config/script/ha-script-picker.ts

```diff
@@ -97,7 +113,9 @@ class HaScriptPicker extends LitElement {
     const script = ev.currentTarget.script as HassEntity;
     await triggerScript(this.hass, script.entity_id);
     showToast(this, {
-      message: `Triggered ${computeStateName(script)}`,
+      message: `${this.hass.localize(
+        "ui.dialogs.notification_toast.triggered"
+      )} ${computeStateName(script)}`,
```

Comment:
Let's give `computeStateName(script)` as variable to lokalize, so the order can be changed by the translator:

```js
${this.hass.localize("ui.dialogs.notification_toast.triggered", "name", computeStateName(script))}
```

The string in `en.json`would be:
`Triggered {name}`